tests/rpm/BUILD

```diff
@@ -179,6 +179,7 @@ pkg_rpm(
     architecture = "noarch",
     conflicts = ["not-a-test"],
     description = """pkg_rpm test rpm description""",
+    epoch = "1",
```

Comment:
Since this is optional, it should not be in every test case.

Reply:
Agree. Updated 😄

Reply:
Agree! Updated :smile: